src/cromshell/status/command.py

```diff
@@ -0,0 +1,118 @@
+import logging
+import click
+
+import requests
+import tempfile
+import json
+import re
+import fileinput
+import jq
+
+from cromshell.utilities import HTTPUtils
+from cromshell.utilities import IOUtils
+from cromshell.utilities import VariableSetup
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="status")
+@click.argument('workflow_id')
+def main(workflow_id):
+    """Check the status of a Cromwell job UUID"""
+    click.echo("status")
+    LOGGER.info("status")
+
+    retVal = 0
+    curl_out = "str"
+    tempf = tempfile.TemporaryFile()
+    # Create an object to hold all cromwell configurations
+    cromwell_config = VariableSetup.Cromwell()
```

Comment:
I think it would be good to have this created in the main file because its an object that would be used by several other commands. But i don't know how to pass the object to the status function?